test/inheritance_test.rb

```diff
@@ -14,10 +14,10 @@ class NewBar < BaseBaseDoc
 
 class InheritanceTest < Test::Unit::TestCase
   test 'property inheritance' do
-    assert_equal(BaseBaseDoc.new.properties.map(&:key), [:supertype, :doc_type])
+    assert_equal(BaseBaseDoc.new.properties.map(&:key), [:supertype, :type])
```

Comment:
can you assert the type value please?

Reply:
@javierg this is already asserted in https://github.com/amco/dolly/pull/151/files#diff-a285168c25485dd4cf9629158a3c0f94R17